packages/frontend/components/primitives/link/styles.ts

```diff
@@ -68,4 +68,9 @@ export const linkStyles = ({
     line-height: ${lineHeights.tub};
     color: ${colors.text};
   }
+
+  :disabled {
```

Comment:
anchor don't have` :disabled pseudo` class. You can specify aria-disabled here and in component itself

Reply:
done